src/SimpleStorageSetup.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.17;
+
+import {PermissionLib} from "@aragon/osx/core/permission/PermissionLib.sol";
+import {PluginSetup, IPluginSetup} from "@aragon/osx/framework/plugin/setup/PluginSetup.sol";
+import {SimpleStorage} from "./SimpleStorage.sol";
+
+/// @title SimpleStorageSetup build 1
+contract SimpleStorageSetup is PluginSetup {
+    address private immutable IMPLEMEMTATION;
+
+    constructor() {
+        IMPLEMEMTATION = address(new SimpleStorage());
+    }
+
+    /// @inheritdoc IPluginSetup
+    function prepareInstallation(
+        address _dao,
+        bytes memory _data
+    ) external returns (address plugin, PreparedSetupData memory preparedSetupData) {
+        uint256 number = abi.decode(_data, (uint256));
+
+        plugin = createERC1967Proxy(
+            IMPLEMEMTATION,
+            abi.encodeWithSelector(SimpleStorage.initialize.selector, _dao, number)
```

Comment:
please create a task to update the examples to use [`abi.encodeCall`](https://solidity-by-example.org/abi-encode/) instead `abi.encodeWithSelector`

Reply:
this is Dope!